<launch>

Per vedere le webcam collegate al pc e il relativo device:
    sudo apt-get install v4l-utils
    v4l2-ctl --list-devices


<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video2" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_info_url" value="file://$(find tasks_utils)/params/camera_params.yaml" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
</node>

<node  name="usb_cam_tf" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 base_link usb_cam 10" />

</launch>